packages/firebase_vertexai/firebase_vertexai/example/lib/pages/bidi_page.dart

```diff
@@ -59,7 +59,7 @@ class _BidiPageState extends State<BidiPage> {
     super.initState();
 
     final config = LiveGenerationConfig(
-      speechConfig: SpeechConfig(voiceName: 'Fenrir'),
+      speechConfig: SpeechConfig(voice: Voice.fenrir),
```

Comment:
This needs to keep as the left one. The api has changed for bidi feature.

Reply:
Ah, I had analysis errors because I wasn't overriding to the local package copy. Fixed.